recipe/meta.yaml

```diff
@@ -99,19 +99,9 @@ outputs:
         # other requirements
         - python
         - {{ pin_compatible('numpy') }}
-        # hmaarrfk 2021/12/28
-        # it seems that pytorch uses some aspects of setuptools
-        # that were removed in version 59.6
-        # I'm not really inclined to patch this out since
-        # this is a pretty easy build time dependency
-        # https://github.com/pytorch/pytorch/issues/69894
-        - setuptools <59.6
+        - setuptools
```

Comment:
Is pytorch still run-depending on setuptools at all now?

Reply:
tbh, I'm really not sure.

Reply:
I remember i did try to audit it:
A grep command finds
```~/git/pytorch [v1.11.0|✔]
17:38 $ grep "import setuptools\|from setuptools" . -R
./tools/generate_torch_version.py:from setuptools import distutils  # type: ignore[import]
./tools/build_pytorch_libs.py:from setuptools import distutils  # type: ignore[import]
./tools/coverage_plugins_package/setup.py:import setuptools  # type: ignore[import]
./benchmarks/operator_benchmark/pt_extension/setup.py:from setuptools import setup
./torch/utils/cpp_extension.py:import setuptools
./torch/utils/cpp_extension.py:from setuptools.command.build_ext import build_ext
./torch/utils/cpp_extension.py:        >>> from setuptools import setup
./torch/utils/cpp_extension.py:        >>> from setuptools import setup
./torch/utils/cpp_extension.py:        from setuptools import distutils
./test/test_spectral_ops.py:from setuptools import distutils
./test/cpp_extensions/no_python_abi_suffix_test/setup.py:from setuptools import setup
./test/cpp_extensions/setup.py:from setuptools import setup
./setup.py:from setuptools import setup, Extension, find_packages
./setup.py:from setuptools.dist import Distribution
./setup.py:import setuptools.command.build_ext
./setup.py:import setuptools.command.install
./setup.py:import setuptools.command.sdist
```

But now, do you really want to ensure that cpp_extensions isn't imported?

Reply:
My understanding from @rgommers in #80 was that the runtime-dependency should have been removed?

Reply:
i understood that unpinning was fine in 1.11.

i did not underatand that he meant to remove it outright

Reply:
I'm referring to the discussion in https://github.com/pytorch/pytorch/issues/71280 and particularly [this](https://github.com/pytorch/pytorch/pull/71902#issuecomment-1036250128) comment.

I don't think the vendoring has happened yet, but based on https://github.com/pytorch/pytorch/pull/71902 we'd be able to replace `setuptools` with `packaging`. Unfortunately however, it [seems](https://github.com/pytorch/pytorch/commits/release/1.11) that this was not backported before 1.11 was cut.

Reply:
so lets keep dependijg on it for now?

Reply:
Fine by me - 1.11.0 is out as-is. If https://github.com/pytorch/pytorch/pull/71902 ends up getting backported, we can still adapt for 1.11.1.